jupyter_notebook/notebookapp.py

```diff
@@ -261,6 +301,33 @@ def init_handlers(self, settings):
         new_handlers.append((r'(.*)', Template404))
         return new_handlers
 
+    def _create_widget_shim_handlers(self):
+        """Creates a handler that shims the loading of widgets
+        JS files for the old /static/widgets/js/*.js path."""
+        # Add RESTful shim for the ipython_widgets
+        try:
+            ipythonwidgets = imp.find_module('ipython_widgets')[1]
+            widget_js = os.path.join(ipythonwidgets, 'static', 'widgets')
+
+            return [(
+                r'/deprecatedwidgets/(.*)', 
+                make_shim_handler(self.log, prepend=DEPRECATED_WIDGET_WARNING), 
```

Comment:
I also want the error to appear in the console- redirect doesn't let me do that.